pow/src/main/java/tech/pegasys/teku/pow/DepositFetcher.java

```diff
@@ -295,7 +296,7 @@ public int hashCode() {
     }
 
     @Override
-    public int compareTo(final BlockNumberAndHash o) {
+    public int compareTo(@NotNull final BlockNumberAndHash o) {
```

Comment:
nit: We don't normally use the `@NotNull` annotations (though they do keep sneaking in).

Reply:
IntelliJ suggests it